python/cuml/tests/test_compose.py

```diff
@@ -197,6 +199,9 @@ def test_make_column_transformer_sparse(sparse_clf_dataset,  # noqa: F811
     assert_allclose(t_X, sk_t_X)
 
 
+@pytest.mark.skipif(sklearn.__version__ >= "1.0",
```

Comment:
Can we either do a `getattr` with the appropriate value based on the sklearn version or create an issue to do this later?

Reply:
I think sklearn 1.0 is old enough that you could drop support for sklearn < 1.0 unless you have specific user constraints that need the older version. With that in mind, I'd write the tests the other way (skip on sklearn < 1.0).